blocks.md

```diff
@@ -756,6 +756,8 @@ Key | Value
 ----|-------
 `{location}` | Location name (exact format depends on the service).
 `{temp}` | Temperature.
+`{apparent}` | Apparent Temperature.
```

Comment:
What about `{feelslike}` instead? Also, it might be worth mentioning that we use the "Australian" method with a link to Wikipedia here, rather than in your code.

Reply:
I was also thinking of maybe providing multiple different heat indexes? Different countries use different methods of calculation and there seems to be little logic in why to use one over another. These are some of the different ones I have found.
https://en.wikipedia.org/wiki/Humidex
https://en.wikipedia.org/wiki/Heat_index
https://en.wikipedia.org/wiki/Wind_chill (this one also has the formula for the Australian method)

Reply:
I think apparent is fine as the other ones could be added later. Clarified in the docs.